### PR TITLE
dependabot updates Mon Jun 12 16:47:01 PDT 2023

### PR DESCRIPTION
**Pull request by vasireddy99**
Bump github.com/aws/aws-sdk-go from 1.44.271 to 1.44.280 in /tools/workflow/cleaner #2132
Bump github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsemfexporter from 0.78.0 to 0.79.0 #2131
Bump github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awsxrayreceiver from 0.78.0 to 0.79.0 #2130
Bump github.com/open-telemetry/opentelemetry-collector-contrib/processor/cumulativetodeltaprocessor from 0.78.0 to 0.79.0 #2129
Bump go.opentelemetry.io/build-tools/dbotconf from 0.7.0 to 0.8.0 in /tools/workflow/linters #2128
Bump github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter from 0.78.0 to 0.79.0 #2127
Bump go.opentelemetry.io/collector from 0.78.2 to 0.79.0 #2126

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,9 +4,9 @@ go 1.19
 
 require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/confmap/provider/s3provider v0.78.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsemfexporter v0.78.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsemfexporter v0.79.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsxrayexporter v0.78.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter v0.78.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter v0.79.0
```
Comment (bryan-aguilar) [inline go.mod:9]: All collector dependencies should be updated to the same version

**Review by bryan-aguilar — CHANGES_REQUESTED** [review]

File: go.mod [hunk 2]
```diff
@@ -3,70 +3,70 @@ module github.com/aws-observability/aws-otel-collector
...
-	go.opentelemetry.io/collector/confmap v0.78.2
-	go.opentelemetry.io/collector/exporter v0.78.2
+	go.opentelemetry.io/collector v0.79.0
+	go.opentelemetry.io/collector/component v0.79.0
+	go.opentelemetry.io/collector/confmap v0.79.0
+	go.opentelemetry.io/collector/exporter v0.79.0
 	go.opentelemetry.io/collector/exporter/loggingexporter v0.78.2
 	go.opentelemetry.io/collector/exporter/otlpexporter v0.78.2
 	go.opentelemetry.io/collector/exporter/otlphttpexporter v0.78.2
 	go.opentelemetry.io/collector/extension/ballastextension v0.78.2
```
Comment (bryan-aguilar) [inline go.mod:55]: Need to be updated.

File: go.mod [hunk 3]
```diff
@@ -3,70 +3,70 @@ module github.com/aws-observability/aws-otel-collector
...
-	go.opentelemetry.io/collector/extension/zpagesextension v0.78.2
+	go.opentelemetry.io/collector/extension/zpagesextension v0.79.0
 	go.opentelemetry.io/collector/featuregate v1.0.0-rcv0012
-	go.opentelemetry.io/collector/processor/batchprocessor v0.78.2
+	go.opentelemetry.io/collector/processor/batchprocessor v0.79.0
 	go.opentelemetry.io/collector/processor/memorylimiterprocessor v0.78.2
```
Comment (bryan-aguilar) [inline go.mod:59]: Needs to be updated

File: testbed/go.mod [hunk 4]
```diff
@@ -4,13 +4,13 @@ go 1.20
 
 require (
 	github.com/aws-observability/aws-otel-collector v0.29.0
```
Comment (bryan-aguilar) [inline testbed/go.mod:6]: This should be updated also even though a replace statement is inside

**Review by bryan-aguilar — CHANGES_REQUESTED** [review]

File: go.mod (hunk 2 above)
Comment (vasireddy99) [reply go.mod]: So, it's basically updating the collector version as a whole even if dependabot PRs or script didn't do it. Go it !

Comment (bryan-aguilar) [reply go.mod]: yes

**Review by bryan-aguilar — APPROVED** [review]

**Pull request merged**